sqlglot/dialects/bigquery.py

```diff
@@ -897,3 +1175,61 @@ def version_sql(self, expression: exp.Version) -> str:
             if expression.name == "TIMESTAMP":
                 expression.set("this", "SYSTEM_TIME")
             return super().version_sql(expression)
+
+        def while_sql(self, expression: exp.While):
+            body = self._sql_statements_block(expression.args.get("body", []))
+            suffix = "\nEND WHILE" if body else "END WHILE"
+            return f"WHILE {self.sql(expression, 'this')} DO\n{body}{suffix}"
+
+        def call_sql(self, expression: exp.Call):
+            call_args = ""
+            if expression.args.get("call_args", []):
+                call_args = ",".join([self.sql(e) for e in expression.args.get("call_args", [])])
```

Comment:
Let's leverage existing logic such as `self.expressions` instead of manually implementing csv generators.

Reply:
This should leave a white space between the commas, i.e. `arg1, arg2, ...` - but that will be handled by `self.expressions` if you use it.